developer/src/kmc-keyboard-info/test/test-keyboard-info-compiler.ts

```diff
@@ -50,4 +52,40 @@ describe('keyboard-info-compiler', function () {
 
     assert.deepEqual(actual, expected);
   });
+
+  // ERROR_FileDoesNotExist (.kmp fileSize)
+
+  it('should generate FileDoesNotExist error if .kmp file does not exist', async function() {
```

Comment:
I have stubbed out fillLanguages(), which allows the whole of the shared directory to go, getting us down to 200k; is that enough?  Part of the problem is that run() does an awful lot of work rather than being broken up into more focused methods.

Reply:
Yep, that sounds good. We haven't focused much on this module, because it is used only internally, really, to generate the .keyboard_info files used on downloads.keyman.com. So `run()` is probably going to stay monolithic sorry!